docs/docker-definitions/foundation/azure-cli.adoc

```diff
@@ -0,0 +1,21 @@
+== EIR Foundation Azure CLI
+
+The Azure CLI is a large application that is used to manipulate resource sin Azure. It is is its own image so that it can be used as a base image for other container images, such as `data`.
```

Comment:
```suggestion
The Azure CLI is a large application that is used to manipulate resources in Azure. It is in its own image so that it can be used as a base image for other container images, such as `data`.
```